src/pages/admin/donate/components/support-options/support-options-form/SupportOptionsForm.tsx

```
@@ -0,0 +1,72 @@
+                <div className="support-options-form not-found" data-testid="support-options-not-found">
+                    <img src={NotFoundIcon} alt={COMMON_TEXT_ADMIN.ALT.NOT_FOUND} />
+                    <p>{DONATE_TEXT.SUPPORT_OPTIONS.NOT_FOUND}</p>
+                    <Button className="btn-add" onClick={handleAddOption} buttonStyle="secondary">
+                        {DONATE_TEXT.SUPPORT_OPTIONS.ADD_FIRST}
+                    </Button>
+                </div>
+            ) : (
+                <div className="support-options-form">
+                    <div className="support-options-form title">{DONATE_TEXT.SUPPORT_OPTIONS.TITLE}</div>
+                    {items.map((item) => (
+                        <SupportOptionItem data={item} onSave={handleSaveOption} onDelete={handleDeleteOption} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)